chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,788 @@
 %-----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Sampling and randomized assignment are two core elements of research design.
-In experimental methods, sampling and randomization directly determine
+
+In this chapter we will show how you can save a lot of time
+and increase the quality of your research by planning your data requirements
+based on your research design.
+There are many published resources about 
+the theories behind different research designs.
+This chapter will instead focus on how the design 
+impacts a project's data requirements.
+We assume you have a working familiarity
+with the research designs mentioned here.
+If needed, you can reference Appendix XYZ,
+where you will find more details 
+and specific references for each design.
+
+Planning data requirements is more than just listing the data your project will need. 
+Planning data requirements includes understanding how to
+link the different data sources your project will use.
+It also involves creating the tools your team can use
+to make sure that understanding is shared across your team.
+The first step of this process is to create master datasets 
+for every unit of observation in your study,
+and to create a data map, 
+detailing how the master datasets are interlinked.
```

Comment:
How about:
The first step of the process is to determine the data needs of the project, based on the research design and measurement framework. The data map diagrams each data source the project will use, the unit of response, frequency of measurement, and the level(s) at which it can be linked to other datasets within the framework.